platform/darwin/src/MLNLoggingConfiguration.mm

```diff
@@ -29,6 +29,9 @@ bool onRecord(EventSeverity severity, Event event, int64_t code, const std::stri
             case EventSeverity::Error:
                 MLNLogError(message);
                 break;
+            default:
+                assert(false);
+                return false;
```

Comment:
Nit-picking, but I think it's better to fall through here because otherwise the erroneous log message will get logged with `platformRecord`.